identity/test/package.scala

```diff
@@ -41,7 +41,7 @@ object I18NTestComponents extends I18nComponents {
 
 class IdentityTestSuite extends Suites(
   new EditProfileControllerTest,
-  new EmailControllerTest,
+//  new EmailControllerTest, /* FIXME: Temporarily disabling this test to unblock Laura */
```

Comment:
😿

Reply:
I'll bring it back right after this PR.

Reply:
They are back: https://github.com/guardian/frontend/pull/18225